City-of-Helsinki/drupal-module-helfi-proxy
# UHF-6865, UHF-7347: Removed RedirectResponseSubscriber because it conflicts with other redirects, like Tunnistamo logout

To test this:

- Setup local proxy: https://github.com/City-of-Helsinki/drupal-helfi-local-proxy for KYMP
- Add `$config['helfi_proxy.settings']['default_proxy_domain'] = 'helfi-proxy.docker.so';` to your `local.settings.php` file
- `composer require drupal/helfi_proxy:dev-UHF-6865`
- `drush cr`

Make sure `helfi-kymp.docker.so` is redirected to proxy domain.
